feedback/templates/minimal.html

```diff
@@ -0,0 +1,28 @@
+{% load staticfiles compress i18n %}
+<!DOCTYPE html>
+
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+        <title>{% block title %}{% trans "Helsinki feedback" %}{% endblock %}</title>
+{% compress css %}
+        <link rel="stylesheet" type="text/x-scss" href="{% static "feedback-styles.scss" %}">
+{% endcompress %}
+    </head>
+
+    <body>
+
+
+        <main>
+            {% block main %}{% endblock %}
+        </main>
+
+
+    <!-- jQuery first, then Popper.js, then Bootstrap JS -->
+    <script src="https://code.jquery.com/jquery-3.2.1.slim.min.js" integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN" crossorigin="anonymous"></script>
```

Comment:
There's a mix of 2-space and 4-space indentation in this file.

Reply:
Fixed.